### DISCUSSION TITLE
Macbook Requirements

### DISCUSSION THREAD

**Discussion by UntrappedLimited** [drwjf/BC4000#3] [Q&A]
Why does the mac app require apple M1 chip? Seems unlikely this game would need the M1?

**Comment by drwjf** [comment]
Thanks. That is a very good point. 
The BC4000 have not used catalyst technology to support Mac platform. I will try to build it in the future version.

**Reply by UntrappedLimited** [reply]
Thanks, I've got it on my phone but would love to download this on my late 2015 macbook air so if that could be supported, that would be amazing!
